test-requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/coopiteasy/cooperative@16.0-mig-cooperator-static-test-helpers#subdirectory=setup/cooperator
```

Comment:
Make sure to delete this commit before merging.

Reply:
@carmenbianca are you sure I should remove this commit before merging ? The corresponding PR is still open https://github.com/OCA/cooperative/pull/86 

(test-requirements here was updated)

Reply:
If you want this merged sooner rather than later, it's OK to merge as-is. I didn't realise this wasn't an OCA repo.

It's still not ideal (we have to remember to remove that line again), but it'll do.